clients/vault/src/oracle/storage/impls.rs

```diff
@@ -427,7 +427,7 @@ mod test {
 			.expect("should have an archive url")
 			.clone();
 		let scp_archive_storage = ScpArchiveStorage(testnet_archive_url);
-		let slot_index = 10;
+		let slot_index = 235001;
```

Comment:
This also has to be a regression. We'd prefer using a lower slot

Reply:
Yes, we [changed](https://github.com/pendulum-chain/spacewalk/issues/538) it after opening this PR.